lib/stones-spec/postcondition.rb

```diff
@@ -13,8 +13,10 @@ class FinalBoardPostcondition
 
     attr_reader :final_board_gbb, :check_head_position, :show_initial_board
 
-    def initialize(final_board, check_head_position, show_initial_board)
-      @final_board_gbb = final_board
+    def initialize(example, check_head_position, show_initial_board)
+      @example = example
+      @final_board_gbb = example.final_board
```

Comment:
If we really need the example here, then I would not add final_board_gbb and title attributes.